### PR TITLE
Update readme and add custom rule file 

### PR DESCRIPTION
**Pull request by bryanwux**

### CODE REVIEW

**Comment by leyao-daily** [comment]
please rebase your commits to one

File: README.md [hunk 1]
```diff
@@ -5,3 +5,128 @@ This is the source code of the [modsecurity-wasm-filter](https://github.com/inte
...
 The plugin is the basic version and the modsecurity rule inside the plugin can be updated by the istio CR `WasmPlugin`. And we support dynamic update of the wasm binary and modsecurity rules based on istio > 1.13. The rule server for OWASP rules and customer rules will be delivered in the future.
+
+
+
+### 1. Build and upload wasm plugin:
+
+First, clone the repository and open into `wasmplugin` folder : 
+
+ ```
+ git clone https://github.com/bryanwux/modsecurity-wasm-filter.git
```
Comment (leyao-daily) [inline README.md:16]: intel repo

File: istio/mod-wasm-deploy.yaml [hunk 2]
```diff
@@ -6,8 +6,8 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app: testapp
-  url: oci://registry.fi.intel.com/xiangdai/envoy-modsecurity-wasm-plugin:0.1.0
+      app: istio-ingressgateway
```
Comment (leyao-daily) [inline istio/mod-wasm-deploy.yaml:9]: can work with inject to ingressgateway now?

File: README.md (hunk 1 above)
Comment (bryanwux) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -5,3 +5,128 @@ This is the source code of the [modsecurity-wasm-filter](https://github.com/inte
...
+
+### 2. Deploy wasm plugin into istio-ingressgateway http filter chain
+
+#### Get binding port for http2
+
+```
+kubectl get svc istio-ingressgateway -n istio-system
+
+NAME                   TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)                                      AGE
+istio-ingressgateway   NodePort   10.99.25.89   <none>        15021:31164/TCP,80:30829/TCP,443:30501/TCP   28d
```
Comment (leyao-daily) [inline README.md:38]: Don't know if the ip is valid. It will be good if it is hiden.

Comment (bryanwux) [reply README.md]: > Don't know if the ip is valid. It will be good if it is hiden.

Makes sense. Fixed.

**Pull request merged**